include/modem/lte_lc.h

```diff
@@ -182,10 +182,30 @@ int lte_lc_power_off(void);
  */
 int lte_lc_normal(void);
 
-/** @brief Function for requesting modem to go to or disable
- * power saving mode (PSM) with default settings defined in kconfig.
+/** @brief Function for setting modem PSM parameters:
+ * requested periodic TAU (RPTAU) and requested active time (RAT)
+ * to be used when psm mode is subsequently enabled using `lte_lc_psm_req`.
  * For reference see 3GPP 27.007 Ch. 7.38.
  *
+ * @param rptau Requested periodic TAU.
+ * @param rat Requested active time
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int lte_lc_psm_param_set(const char *rptau, const char *rat);
+
+/** @brief Function for requesting modem to enable or disable
+ * power saving mode (PSM) using default Kconfig value or as set using
+ * `lte_lc_psm_param_set`.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int lte_lc_psm_param_set(const char *req_rptau, const char *req_rat);
```

Comment:
Seems like this function declaration got duplicated by accident, the first one looks like correct one.